src/app/charts/core/files/secret/tpl/arkcase-server.yaml

```diff
@@ -139,9 +139,15 @@ acm.websockets:
     truststore_pass: "${JAVA_TRUSTSTORE_PASS}"
 
 acm.hazelcast:
-  log.folder: "/app/logs/"
+  log.folder: "${LOGS_DIR}"
   # Temporarily disable hazelcast diagnostics
   diagnostics.enabled: false
+  network:
+    port: 5710
+    port.auto_increment: false
+  kubernates:
+    namespace: {{ $.Release.Namespace | quote }}
+    serviceName: {{ include "arkcase.service.headless" $ | quote }}
```

Comment:
You can use `"${POD_NAMESPACE}"` and `"${KUBERNETES_SERVICE_HEADLESS}"` here ... let's avoid templating variables where possible.

Eventually we want our configs to be fixed text files that don't need to be generated at deployment time.